ambry-frontend/src/main/java/com.github.ambry.frontend/AmbryBlobStorageService.java

```diff
@@ -735,9 +735,10 @@ public void onCompletion(Void result, Exception exception) {
             restRequest.getMetricsTracker().injectMetrics(requestMetrics);
           }
           headCallback.markStartTime();
-          router.getBlob(convertedId, new GetBlobOptionsBuilder().operationType(GetBlobOptions.OperationType.BlobInfo)
-              .getOption(getOption)
-              .build(), headCallback);
+          router.getBlob(convertedId,
```

Comment:
just a formatting change? (why?)

Reply:
Forgot to format after changing this back from `BlobInfoAll`